bin/snapshot-backups/downsync-cms

```diff
@@ -82,7 +98,12 @@ echo  aws s3 sync --only-show-errors $WEB_FILES s3://$S3_BUCKET/cms/public/ --ac
 echo cf ssh cms -c ". /etc/profile; drush cr; drush updatedb --no-cache-clear -y; drush cim -y || drush cim -y; drush cim -y; drush php-eval 'node_access_rebuild();' -y"
 $echo cf ssh cms -c ". /etc/profile; drush cr; drush updatedb --no-cache-clear -y; drush cim -y || drush cim -y; drush cim -y; drush php-eval 'node_access_rebuild();' -y"
 
+if [ x$TOME_DISABLED_STATE = x0 ]; then
```

Comment:
It turns out we're deleting the usagov.tome_run_disabled variable rather than setting it to 0 (which is probably why the other tests in this changeset check for 1). 
So maybe what's wanted here is 
`if [x$TOME_DISABLED_STATE != x1 ]` ?